`prep-git` step fails if `yarn` is already installed

<!-- Thank you for contributing. These HTML comments will not render in the issue, but you can delete them once you've read them if you prefer! -->

<!-- Before creating a new issue, please search for relevant issues.
-->

## Description

Trying out the CLI locally, it looks like the `jupyter-releaser prep-git` command fails if `yarn` is already installed.

<!--Describe the bug clearly and concisely. Include screenshots if possible-->

## Reproduce

Create a new conda environment with `yarn`.

Locally, install `jupyter-releaser` from pip with `pip install jupyter-releaser`.

Then run `jupyter-releaser prep-git`.

```
+ git checkout main
Already on 'main'
Your branch is up to date with 'origin/main'.
+ npm install -g yarn
npm ERR! code EEXIST
npm ERR! path /path/to/env/bin/yarn
npm ERR! EEXIST: file already exists
npm ERR! File exists: /path/to/env/bin/yarn
npm ERR! Remove the existing file and try again, or run npm
npm ERR! with --force to overwrite files recklessly.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2021-06-28T13_58_10_269Z-debug.log
```

<!--Describe how you diagnosed the issue-->

## Expected behavior

Either it force replaces the existing yarn, or skip the step if yarn is already installed.

--- a/jupyter_releaser/lib.py
+++ b/jupyter_releaser/lib.py
@@ -486,7 +486,8 @@ def prep_git(ref, branch, repo, auth, username, url, install=True):
 
         # prefer yarn if yarn lock exists
         elif util.YARN_LOCK.exists():
-            util.run("npm install -g yarn")
+            if not shutil.which("yarn"):
+                util.run("npm install -g yarn")
             util.run("yarn")
 
         # npm install otherwise

